package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@lifeomic/abac",
-  "version": "4.3.1",
+  "version": "4.4.0",
```

Comment:
It looks like you will need to push a tag to really publish. Might be nice to bump the version and push the tag in the same push to keep things consistent.

Reply:
I think our OSS packages are still doing this directly in `master` after the PR merge.